package.json

```diff
@@ -36,14 +36,16 @@
     "@babel/register": "^7.5.5",
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
-    "cross-env": "^5.2.0",
     "coveralls": "^3.0.6",
+    "cross-env": "^5.2.0",
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.2.0",
     "eslint-plugin-import": "^2.18.2",
     "mocha": "^6.2.0",
     "mocha-lcov-reporter": "^1.3.0",
     "nodemon": "^1.19.1",
-    "nyc": "^14.1.1"
+    "nyc": "^14.1.1",
+    "sequelize-test-helpers": "^1.1.2",
```

Comment:
@sa-ma are you still using sequelize-test-helpers??

Reply:
Yeah we are